pkg/cmd/create.go

```diff
@@ -81,12 +85,28 @@ func newCreateCmd(out io.Writer) *cobra.Command {
 	}
 
 	cmd.Flags().StringVarP(&o.starter, "starter", "p", "", "the name or absolute path to Helm starter scaffold")
+	cmd.Flags().StringVar(&o.chartAPIVersion, "chart-api-version", chart.APIVersionV2, "chart API version to use (v2 or v3)")
```

Comment:
Can this flag be hidden when the `HELM_EXPERIMENTAL_CHART_V3` experiment is not enabled.